language/stdlib/modules/LibraAccount.move

```diff
@@ -1040,6 +1065,48 @@ module LibraAccount {
         make_account(lr_account, auth_key_prefix)
     }
 
+    spec fun create_libra_root_account {
+        pragma opaque;
+        include CreateLibraRootAccountModifies;
+        include CreateLibraRootAccountAbortsIf;
```

Comment:
Maybe instead of having all the `aborts_if exists<Resource>`s here, we should have a predicate in `Genesis` saying that before genesis no resource exists at any address?

Reply:
That is an interesting question. One problem is that there are intermediate states after the initial empty state anyway. And in fact, someone may call accidentally initialize functions twice.